src/libValidator/Validator.cpp

```diff
@@ -0,0 +1,241 @@
+/**
+* Copyright (c) 2018 Zilliqa 
+* This source code is being disclosed to you solely for the purpose of your participation in 
+* testing Zilliqa. You may view, compile and run the code for that purpose and pursuant to 
+* the protocols and algorithms that are programmed into, and intended by, the code. You may 
+* not do anything else with the code without express permission from Zilliqa Research Pte. Ltd., 
+* including modifying or publishing the code (or any part of it), and developing or forming 
+* another public or private blockchain network. This source code is provided ‘as is’ and no 
+* warranties are given as to title or non-infringement, merchantability or fitness for purpose 
+* and, to the extent permitted by law, all liability for your use of the code is disclaimed. 
+* Some programs in this code are governed by the GNU General Public License v3.0 (available at 
+* https://www.gnu.org/licenses/gpl-3.0.en.html) (‘GPLv3’). The programs that are governed by 
+* GPLv3.0 are those programs that are located in the folders src/depends and tests/depends 
+* and which include a reference to GPLv3 in their program files.
+**/
+
+#include <vector>
+
+#include "Validator.h"
```

Comment:
alpbaetical order

Reply:
I am not sure what you mean here.
It seems like in most files (all the ones I checked at least), includes are sorted in the following order:
1. Standard headers (alphabetical order)
2. Class header (matching .h file for a .cpp file)
3. Project headers (alphabetical order)

It is also what is is written in the wiki. Can you please clarify which convention you would like to follow ?

Reply:
resolved. no changed required